src/main/scala/util/package.scala

```diff
@@ -1,6 +1,24 @@
 import java.io.{File, PrintWriter}
+import java.util.concurrent.{Executors, TimeUnit, TimeoutException}
 
 package object util {
+
+  // WARNING: does not kill the thread, we only return to host program
+  // Thread.stop() is deprecated and cannot be used here
```

Comment:
is this good enough when autotuning?

Reply:
It is okay when the problem of generating tons of kernel-code only occurs a few times during a tuning run. We should definitely find a solution for that.

Reply:
Maybe you can open an issue to track this then?

Reply:
where is the issue?

Reply:
Issue is open